src/v1/doc/google/protobuf/doc_field_mask.js

```diff
@@ -84,57 +84,49 @@
  * describe the updated values, the API ignores the values of all
  * fields not covered by the mask.
  *
- * If a repeated field is specified for an update operation, the existing
- * repeated values in the target resource will be overwritten by the new values.
- * Note that a repeated field is only allowed in the last position of a `paths`
- * string.
+ * If a repeated field is specified for an update operation, new values will
```

Comment:
@JustinBeckwith @jkwlui this looks like a breaking change ⚠️ Any thoughts about how to handle this?

Reply:
eesh, this is a weird one since it looks like it's just a change in the proto comment, but the behavior may have changed?  @jskeet would you mind poking at this and seeing what's up?

Reply:
This comes from https://github.com/protocolbuffers/protobuf/blob/master/src/google/protobuf/field_mask.proto
The [change](https://github.com/protocolbuffers/protobuf/commit/6bbe197e9c1b6fc38cbdc45e3bf83fa7ced792a3#diff-9ed9033c8c6185a02f2d6d173b5b4beb) was a massive "integrate from google3" commit, so we've lost history of why it's happened - I'll look internally.

Reply:
From an internal thread - it looks like this was a doc fix, and now the comment actually covers the historical behavior.